README.md

```
@@ -1,2 +1,101 @@
+
+![n8n Workflow Screenshot](n8n_workflow.png)
+
+(*Animated GIFs or short videos can be added here if available.*)
+
+---
+
+## 📖 Detailed Documentation
+
+- [Detailed Gamma.ai Documentation](https://gamma.app/your-onboarding-documentation)
+
+*(Replace the above placeholder link with your Gamma.ai documentation.)*
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)